### PR TITLE
dxFileUploader: add HowTo section

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Use the [uploadHeaders](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadHeaders) option to specify a header to the request.
```
Comment (dirk-pieterse) [inline concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md]: How can you "specify a header to the request"? What it mean when you 'specify' a header to a request?

File: concepts/05 Widgets/FileUploader/99 How To/20 Specify a file's GUID.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
+Use the [valueChanged](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Events/#valueChanged) event to generate a unique identifier for a file before uploading and pass it as a parameter to an upload URL.
```
Comment (dirk-pieterse) [inline concepts/05 Widgets/FileUploader/99 How To/20 Specify a file's GUID.md]:
```suggestion
Use the [valueChanged](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Events/#valueChanged) event to generate a unique identifier for a file before it is uploaded and pass it as a parameter to the upload URL. 
```

File: concepts/05 Widgets/FileUploader/99 How To/30 Get a file's Guid after uploading in chunks.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+Follow the steps below to get a file's GUID in 'chunk' [upload mode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode):
+
+1. Get the FileGuid value from the chunkMetadata parameter on the server side.
```
Comment (dirk-pieterse) [inline concepts/05 Widgets/FileUploader/99 How To/30 Get a file's Guid after uploading in chunks.md]:
```suggestion
1. Get the FileGuid value from the chunkMetadata parameter on the server.
```

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md (hunk 1 above)
Comment (AnnaKondratova) [reply concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md]: [Request headers](https://developer.mozilla.org/en-US/docs/Glossary/Request_header) contain additional information passed with a request.

Request header has the following format: {header_name) : {header value}

For example request headers for a request:

Host: developer.mozilla.org
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:50.0) Gecko/20100101 Firefox/50.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-US,en;q=0.5 
// Added a new header
my-own-header: test-value

"Specify a header for a request" is an usual phrase in this context.

Comment (dirk-pieterse) [reply concepts/05 Widgets/FileUploader/99 How To/10 Specify a Header to the Request.md]: Specify 'for', but not 'to'

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**